src/ispc.cpp

```diff
@@ -793,10 +793,17 @@ Target::Target(Arch arch, const char *cpu, ISPCTarget ispc_target, bool pic, boo
               target_string.c_str());
         return;
     }
+#ifdef ISPC_XE_ENABLED
```

Comment:
Let's have {} around the if statement body and you don't need ifdef, as `ISPCTargetIsGen()` function is generally available.